pom.xml

```diff
@@ -31,13 +31,13 @@
         <munit.input.directory>src/test/munit</munit.input.directory>
         <munit.citizen.input.directory>src/test/munit-citizen</munit.citizen.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
-        <munit.extensions.maven.plugin.version>1.2.0-SNAPSHOT</munit.extensions.maven.plugin.version>
-        <munit.version>3.1.0-SNAPSHOT</munit.version>
+        <munit.extensions.maven.plugin.version>1.2.0-rc4</munit.extensions.maven.plugin.version>
+        <munit.version>3.1.0-rc5</munit.version>
 
         <mtf.javaopts></mtf.javaopts>
 
         <!-- runtime version to run -->
-        <runtimeVersion>4.6.0-SNAPSHOT</runtimeVersion>
+        <runtimeVersion>4.6.0-rc4</runtimeVersion>
```

Comment:
I'm not sure why this pom originally indicated an explicit runtimeVersion. That must happen through a build.yaml config or indicating the runtime version on the branch build on Jenkins. As a general rule all versions from 4.3.0 should be ran for this code and each JVM except J17. Please remove this property and the `<runtimeVersion>${runtimeVersion}</runtimeVersion>` tag making use of it in the configuration.